src/convertToObject.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
   // write your code here
+  const cssParams = {};
+  // write your code here
+  const splitByNewLine = sourceString.split(';');
+
+  for (let i = 0; i < splitByNewLine.length; i++) {
```

Comment:
good job.
let's replace for loop on forEach or better on reduce with initial value {}